index.html

```diff
@@ -1,57 +1,230 @@
-           Glad to see you!</h1>
+            Glad to see you!</h1>
           <p id="description">
-          I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me. 
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
           </p>
           <h2 id="connect" class="connect-title">LET’S CONNECT</h2>
-         
+
         </div>
-        </section>
-        <section id="icons-secction">
-          <ul class="social-media-icons">
-      <li class="icons">
-        <a href="#" target="_blank"><img src="img/twitter.png" class="img-icons"></a>
-      </li>
-       <li class="icons">
-        <a href="#" target="_blank"><img src="img/Linkedin icon.png" class="img-icons"></a>              
-      </li>
-      <li class="icons">
-        <a href="#"><img src="img/M.png" class="img-icons"></a>             
-      </li>
-      <li class="icons">
-        <a href="#" target="_blank"><img src="img/GitHub.png" class="img-icons"></a>              
-      </li>
-      <li class="icons">
-        <a href="#"><img src="img/Paz.png" class="img-icons"></a>
-      </li>
-         
-      </ul>
       </section>
+      <section id="icons-secction">
+        <ul class="social-media-icons">
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/twitter.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/Linkedin icon.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#"><img src="img/M.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#" target="_blank"><img src="img/GitHub.png" class="img-icons"></a>
+          </li>
+          <li class="icons">
+            <a href="#"><img src="img/Paz.png" class="img-icons"></a>
+          </li>
+
+        </ul>
+      </section>
+
+
+    </div>
+  </div>
+  <div class="Works">
+    <section class="screen-shot ">
+      <img src="img/Snapshoot Portfolio.png" alt="project-image" class="screenshot-img">
+    </section>
+    <section class="text-block">
+      <header class="header-3">
+        <h2 class="project-tittle">Tonic</h2>
+      </header>
+      <div>
+        <ul class="frame">
+          <li class="txt" id="canopy">CANOPY</li>
+          <li class="img-container"><img src="img/Counter.png" alt="point" id="img-point"></li>
+          <li class="txt" id="role">Back End Dev</li>
+          <li class="img-container"><img src="img/Counter.png" alt="point" id="img-point-2"></li>
+          <li class="txt" id="year">2015</li>
+        </ul>
+      </div>
+      <div>
```

Comment:
[OPTIONAL] AS mentioned before please be sure to add all cards on the works section 👍